src/Uno.UI/UI/Xaml/Controls/TimePicker/NativeTimePickerFlyout.Android.cs

```diff
@@ -13,7 +13,7 @@ partial class NativeTimePickerFlyout
 {
 	private bool _programmaticallyDismissed;
 	private UnoTimePickerDialog _dialog;
-	private TimeSpan _initialTime;
+	public TimeSpan _initialTime;
```

Comment:
I don't think we should change the visibility here. I believe the only reason was so you could access it during testing?